apophis/cmds/define.py

```diff
@@ -4,24 +4,28 @@
 
 class DefineCommand(Command):
     def handle(self, context, message):
-        content = list(message.content[8:].split())
-        last = content[-1]
+        content = message.content[8:].split()
+
+        try:
+            index = int(content[-1])
+            term = ' '.join(content[:-1])
+        except ValueError:
+            index = 1
+            term = ' '.join(content)
```

Comment:
can you use `str.join` here instead so it's readily clear what's happening, it's `str#join` is not 'recommended in python 3' it's just available for use.

Reply:
Done :)